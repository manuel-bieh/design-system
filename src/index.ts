import './assets/css/index.css';
import './assets/css/fonts.css';

export * from './Components/ClientLogos';
export * from './Layout/Header';
export * from './Layout/Grid/Grid';
export * from './Layout/Grid/Column';
