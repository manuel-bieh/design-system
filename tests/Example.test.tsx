import React from 'react';
import renderer from 'react-test-renderer';
import { expect, it } from 'vitest';
import { Header } from '../src';

it('renders correctly', () => {
    const tree = renderer.create(<Header>Test</Header>).toJSON();
    expect(tree).toMatchSnapshot();
});
