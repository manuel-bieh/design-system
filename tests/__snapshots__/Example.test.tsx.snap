// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders correctly 1`] = `
<header
  className="_wrapper_f67eb6"
>
  Test
</header>
`;
